### ISSUE TITLE
Adjusting --add_package 's description?

### ISSUE THREAD

**Issue by rayluo** [brython-dev/brython#1605]
I'm currently still new to Brython and learning different concepts. This `--add_package` command line option ~does not seem to be mentioned anywhere in the [official documentation](https://brython.info/static_doc/en/install.html), so, the only info for a new comer like me is from command-line tool~ UPDATE: it is documented in the section "Installing a CPython package" of [that documentation page](https://brython.info/static_doc/en/install.html). But still, I would hope the command-line description be more descriptive, so that new comers do not have to RTFM. :-)

The current command-line help:

```
brython-cli -h
...
  --add_package ADD_PACKAGE
                        Add a CPython package in Lib/site-packages
...
```

At first glance, I thought it would do the same work as `pip install`. After some trial run and discovering [the last comment of this conversation thread](https://github.com/brython-dev/brython/issues/635#issuecomment-569416750):

> ... what you can do is to install CPython packages with pip, then add them to a Brython application by `python -m brython add_package <package_name>`.

I would suggest to change its description into something like this.

```
  --add_package ADD_PACKAGE
                        Add an already-installed, pure Python package from current environment
                        into current project's ./Lib/site-packages
                        so that you can deploy it with your Brython application
```

If needed, I can submit a pull request for this change. Just let me know.

**Comment by PierreQuentel** [comment]
@rayluo 
You are right, this would be more clear. I would rephrase it slightly differently

>                        Add an already-installed, pure-Python package from current CPython environment
>                        into current Brython project's ./Lib/site-packages

I will merge the PR as soon as you submit it. Thanks !
